build_n_deploy/naiserator/nais-prod.yaml

```diff
@@ -29,6 +29,8 @@ spec:
     sidecar:
       enabled: true
       autoLogin: true
+      autoLoginIgnorePaths:
+        - /familie/alene-med-barn/ettersending/static/**/*.css
```

Comment:
pathen for ettersending er uten static :)

Reply:
Hvordan ser man det?